ENIGMAsystem/SHELL/Widget_Systems/xlib/Info/About.ey

```diff
@@ -1,15 +1,16 @@
 %e-yaml
 ---
 
-Name: Linux X11 Widgets
+Name: Linux/FreeBSD X11 Widgets
```

Comment:
no need to list OS in name as osx and likely other oses can run xlib stuff too

Reply:
Yeah I'll make it just say "X11" and "X11 Widgets" for the related Platform and Widget System

Reply:
I actually think I meant to do this change in my BSD pr